internal/models/query.go

```diff
@@ -9,20 +9,21 @@ type QueryResourceModel struct {
 }
 
 type QuerySpecificationModel struct {
-	ID                types.String                             `tfsdk:"id"`
-	FilterCombination types.String                             `tfsdk:"filter_combination"`
-	Breakdowns        []types.String                           `tfsdk:"breakdowns"`
-	Limit             types.Int64                              `tfsdk:"limit"`
-	TimeRange         types.Int64                              `tfsdk:"time_range"`
-	StartTime         types.Int64                              `tfsdk:"start_time"`
-	EndTime           types.Int64                              `tfsdk:"end_time"`
-	Granularity       types.Int64                              `tfsdk:"granularity"`
-	Calculations      []QuerySpecificationCalculationModel     `tfsdk:"calculation"`
-	CalculatedFields  []QuerySpecificationCalculatedFieldModel `tfsdk:"calculated_field"`
-	Filters           []QuerySpecificationFilterModel          `tfsdk:"filter"`
-	Havings           []QuerySpecificationHavingModel          `tfsdk:"having"`
-	Orders            []QuerySpecificationOrderModel           `tfsdk:"order"`
-	Json              types.String                             `tfsdk:"json"` // Computed JSON query specification output
+	ID                       types.String                             `tfsdk:"id"`
+	FilterCombination        types.String                             `tfsdk:"filter_combination"`
+	Breakdowns               []types.String                           `tfsdk:"breakdowns"`
+	Limit                    types.Int64                              `tfsdk:"limit"`
+	TimeRange                types.Int64                              `tfsdk:"time_range"`
+	StartTime                types.Int64                              `tfsdk:"start_time"`
+	EndTime                  types.Int64                              `tfsdk:"end_time"`
+	Granularity              types.Int64                              `tfsdk:"granularity"`
+	CompareTimeOffsetSeconds types.Int64                              `tfsdk:"compare_time_offset_seconds"`
```

Comment:
I wonder if the attribute should just be named `compare_time_offset` for the resource? It's fine in the API and the client, but the `_seconds` feels unnecessary to me especially given that `granularity` is also in seconds and we don't have the suffix there